int32 robot_id
PointFloor position
---
bool ack